lib/client/jsx/components/model_map.jsx

```diff
@@ -93,8 +93,8 @@ class LayoutNode {
     let template = this.template
     this.links = Object.keys(template.attributes).map((att_name) => {
       let attribute = template.attributes[att_name]
-      if (!attribute.model_name) return null
-      let other = this.layout.nodes[ attribute.model_name ]
+      if (!attribute.link_model_name) return null
+      let other = this.layout.nodes[ attribute.link_model_name ]
       if (!other) return null
 
       // the link exists if - you are the other model's parents
```

Comment:
Do the next couple of lines also need to change to `link_model_name`?

Reply:
👍